include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -191,15 +181,15 @@ struct buffered_piece_collection
         point_type m_center;
 
         piece()
-            : type(strategy::buffer::piece_type_unknown)
-            , index(-1)
-            , left_index(-1)
-            , right_index(-1)
-            , beyond_last_segment_index(-1)
-            , offsetted_count(-1)
-            , is_flat_start(false)
-            , is_flat_end(false)
-            , is_deflated(false)
+            : type(strategy::buffer::piece_type_unknown),
+              index(-1),
+              left_index(-1),
+              right_index(-1),
+              beyond_last_segment_index(-1),
+              offsetted_count(-1),
+              is_flat_start(false),
+              is_flat_end(false),
+              is_deflated(false)
```

Comment:
This is also changed, I cannot find how to leave the comma before.
Also here, maybe we can live with it

Reply:
Fixed using `BreakConstructorInitializers`